src/app/components/list/overview/index.html

```diff
@@ -75,6 +75,180 @@
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSortFieldSelectorModel properties">
+    <sky-alert alertType="warning">The following properties are a work in progress and not recommended for use. </sky-alert>
+    <sky-demo-page-property
+      propertyName="descending"
+      isOptional="true"
+      defaultValue="false"
+    >
+    Indicates whether to sort in descending order. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fieldSelector"
+      isOptional="true"
+    >
+    Specifies the fields to sort. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSortLabelModel properties">
+    <sky-alert alertType="warning">The following properties are a work in progress and not recommended for use.</sky-alert>
+    <sky-demo-page-property
+      propertyName="fieldType"
+      isOptional="true"
+    >
+    Specifies the label type. This property accepts <sky-code>string</sky-code>values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fieldSelector"
+      isOptional="true"
+    >
+    Specifies the fields to sort. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="global"
+      isOptional="true"
+      defaultValue="false"
+    >
+    Specifies all fields to sort. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="descending"
+      isOptional="true"
+      defaultValue="false"
+    >
+    Indicates whether to sort in descending order. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="text"
+      isOptional="true"
+    >
+    Specifies the text for the label. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSortModel properties">
+    <sky-demo-page-property
+      propertyName="available"
+      isOptional="true"
+    >
+    Specifies the list of available views to sort. This property accepts an array of <sky-code> ListSortLabelModel</sky-code> objects.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fieldSelectors"
+      isOptional="true"
+    >
+    Specifies the fields to sort. This property accepts an array of <sky-code>ListSortFieldSelectorModel</sky-code> objects.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="global"
+      isOptional="true"
+    >
+    Specifies the fields to sort. This property accepts an array of <sky-code>ListSortLabelModel</sky-code> objects.
```

Comment:
Should we link `ListSortLabelModel` and `ListSortFieldSelectorModel ` to their respective tables above?